meson.build

```diff
@@ -61,6 +61,34 @@ executable(
 	],
 	include_directories: [inc],
 	install: true,
+	install_dir: get_option('libexecdir'),
+)
+
+conf_data = configuration_data()
+conf_data.set('libexecdir',
+	join_paths(get_option('prefix'), get_option('libexecdir')))
+conf_data.set('systemd_service', '')
+
+systemd = dependency('systemd', required: get_option('systemd'))
+
+if systemd.found()
```

Comment:
Can we move all of this logic into `contrib/systemd/meson.build` and just call `subdir` here?

Reply:
Content of D-Bus service file depends on systemd.found() so I'm a bit unsure how to reorganize that. I can move all service files under contrib and create meson.build there.

Note: dependency is taken from a strong wording in [dbus-daemon(1)](https://dbus.freedesktop.org/doc/dbus-daemon.1.html):
>  Services that declare the optional `SystemdService` must also provide a systemd user service unit file whose name or `Alias` matches the `SystemdService`

Reply:
Eh, I see. Then you're right, let's just keep it here for now.